src/main/java/com.gigssandbox/command/CommandFactory.java

```diff
@@ -39,10 +39,18 @@ public static Command createCommandFrom(String... strings) {
     }
 
     private static CommandType verifyParametersForCommand(String[] strings, CommandType commandType) {
```

Comment:
this method becomes larger and introduces new validation responsibility. need to create CommandValidator class.